### DISCUSSION TITLE
some clarification questions

### DISCUSSION THREAD

**Discussion by etinaude** [rdudhagra/Sand-Table#17] [Q&A] [answered]
Heya, 

I'm planning on making this table but I have a  few questions.

1. what's the use of the uno? cant the LEDS be run off of the PIs GPIO?
2. I see there are noise dampers for the motors, did they make much noise without them?
3. on some other resources I heard that the ball rolling made a loud noise, did you find this?
4. what did you find with magnet size, did this one ever slip? 

your table looks amazing btw! And really well documented

**Comment by rdudhagra** [comment, marked as answer]
Hey, glad you found this helpful!

1. Yep, you can pretty easily run the LED's off of the Pi. I had the UNO on hand and it made things a bit simpler, but you could easily power them off the Pi if you want.
2. The steppers on their own are pretty quiet. However, in this case, they're attached to a large aluminum frame, which is attached to a large wooden box...aka the sand table acts like an amplifier and ends up making the steppers sound way louder. So, the dampers definitely help in this case. I also isolated the gantry from the table using some packing foam, but no idea if that actually helps or not. Also note that the table does become quieter once you put the glass top onto it.
3. Yea, I've experimented with this a bit. The trick is that the surface underneath the ball needs to have some "give" to it. I used a canvas-like cloth on top of the wood and I found that to help a lot. Also (as I mentioned earlier) the ball becomes much quieter once you put the glass on top, but definitely don't just put the ball directly on the wood platform like some of the other guides, since it'll be loud as you said.
4. The 1 inch cube neodymium magnet is what I had on hand, and it seems to work pretty well. I've seen people get away with smaller magnets, but their platform is a lot thinner. I used 1/2" plywood for that, but you can get away with 1/4" pretty easily, which should make the magnet much more reliable. It took a lot of tuning for me to get the magnet at the correct height where the ball wouldn't slip; however, it's very reliable now that it's tuned. I think as long as you have a reasonably strong magnet you'll be fine (and use 1/4" plywood instead of 1/2" for the platform base if you want to be extra-sure). You'll probably run into the issue where one corner of the platform is higher than another corner rather than the entire platform being too low/high, so have some shims/etc on hand to make it level.

Let me know if that answers your questions, and feel free to follow up with anything else. Good luck!

**Reply by etinaude** [reply]
Great! Thanks so much!
